### PR TITLE
dev, v3.0: add the doc of TiDB FAQs in Kubernetes

### PR DESCRIPTION
**Pull request by CaitinChen**
### What is changed, added or deleted? 

Add the doc of TiDB FAQs in Kubernetes.

### What is the related PR or file link(s)? 

https://github.com/pingcap/docs-cn/pull/1517

### Which version does your change affect? 

dev, v3.0

### Checklist 

- [x] Add a new file to `TOC.md`
- [x] No Tab spaces anywhere 
- [x] Leave a blank line both before and after a code block
- [x] Keep the first level heading consistent with `title` in metadata
- [x] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

File: dev/faq/tidb-in-kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: TiDB FAQs in Kubernetes
+summary: Learn about TiDB FAQs in Kubernetes.
+category: FAQ
```
Comment (DanielZhangQD) [inline dev/faq/tidb-in-kubernetes.md]: The change in https://github.com/pingcap/docs-cn/pull/1601 will be addressed in another PR?

Comment (CaitinChen) [reply dev/faq/tidb-in-kubernetes.md]: @danielzhangQD These updates will be added to the troubleshooting section.

Comment (CaitinChen) [reply dev/faq/tidb-in-kubernetes.md]: @danielzhangQD These updates will be added to the troubleshooting section.

**Comment by CaitinChen** [comment]
@dcalvin Thanks. PTAL again.
